examples/lusid/fx-forwards/create-fx-forward.sql

```diff
@@ -0,0 +1,37 @@
+-- ===============================================================
+-- Description:
+-- 1. In this query, we run an ETL process on a Fx Forward.
```

Comment:
Very (very) minor thing, as only 1, can it say 'an fx forward...'

Reply:
My English skills letting me down....